### PR TITLE
Fixed issue #15904: Default survey admin & admin email not copied from user profile

### PR DESCRIPTION
**Pull request by gabrieljenik**
Allow user to select who is the admin of the newly created survey.
Had to adapt the CreateSurvey services and related to be able to take survey_admin as a parameter

### CODE REVIEW

File: application/datavalueobjects/SimpleSurveyValues.php [hunk 1]
```diff
@@ -25,6 +25,12 @@ class SimpleSurveyValues
     /** @var  int the surveygroup from which the new survey will inherit values */
     private $surveyGroupId;
 
+    /** @var string administrator name */
+    private $admin = 'inherit';
```
Comment (olleharstedt) [inline application/datavalueobjects/SimpleSurveyValues.php:29]: There's no need to make this private if you have both a (trivial) getter and setter, I think.

File: application/datavalueobjects/SimpleSurveyValues.php [hunk 2]
```diff
@@ -25,6 +25,12 @@ class SimpleSurveyValues
...
+
+    /** @var string administrator email */
+    private $adminEmail = 'inherit';
```
Comment (olleharstedt) [inline application/datavalueobjects/SimpleSurveyValues.php:32]: Same here

**Review by olleharstedt — CHANGES_REQUESTED** [review]

**Review by gabrieljenik — COMMENTED** [review]
Updated code

**Comment by olleharstedt** [comment]
Thanks!

Now go prepare your arguments for why getters/setters are always a good idea, even when trivial. :D

**Comment by gabrieljenik** [comment]
> Now go prepare your arguments for why getters/setters are always a good idea, even when trivial. :D

I know which battles to avoud! haha

**Comment by gabrieljenik** [comment]
> │ Error: Call to undefined method LimeSurvey\Datavalueobjects\SimpleSurveyValues::setBaseLanguage()
> 
> │
> 
> │ /home/travis/build/LimeSurvey/LimeSurvey/tests/unit/models/CreateSurveyServiceTest.php:44

I knew I should have stood stronger on that battle!

**Comment by gabrieljenik** [comment]
> │ Error: Call to undefined method LimeSurvey\Datavalueobjects\SimpleSurveyValues::setBaseLanguage()
> 
> │
> 
> │ /home/travis/build/LimeSurvey/LimeSurvey/tests/unit/models/CreateSurveyServiceTest.php:44

Let's see how travis reacts now

**Pull request merged**